src/Admin/Redirect.php

```diff
@@ -82,6 +82,28 @@ public function activate(): void {
 		}
 	}
 
+	/**
+	 * Check if onboarding is complete.
+	 * For backwards compatibility, checks MC setup if ONBOARDING_COMPLETED_AT is not set.
+	 *
+	 * @return bool
+	 */
+	protected function is_onboarding_complete(): bool {
```

Comment:
Thanks @jamesmorrison there is a new requirement to add this value to the `glaData`.

To do this I think we should move this method to the `OnboardingCompleted` class and reference it from there.

We can use the `MerchantCenterAwareInterface` and `MerchantCenterAwareTrait` so the service is available in `OnboardingCompleted`.

I've updated the ticket to reflect this.